### ISSUE TITLE
Feat: Add Login and sign up page 

### ISSUE THREAD

**Issue by sakshiigdtuw** [EternoSeeker/gameoflife#288]
### Describe the feature

I want to add login and signup page for the users in game . 
Kindly assign me this Issue.

### Add ScreenShots

![Screenshot 2024-05-29 200818](https://github.com/EternoSeeker/gameoflife/assets/154981554/674787c8-0baf-4702-ac99-1d8ca983d5a9)
![Screenshot 2024-05-29 200142](https://github.com/EternoSeeker/gameoflife/assets/154981554/397bd2b4-d237-45fc-8c28-2e88bcc369c4)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Sukhandeepkaur769** [comment]
yes i can do it I am a GSSOC'24 contributor.   I want to work on this issue
